pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -43,8 +45,9 @@ import (
 )
 
 const (
-	podInterface = "eth0"
-	guestDNS     = "8.8.8.8"
+	podInterface  = "eth0"
+	guestDNS      = "8.8.8.8"
```

Comment:
Something bugs me about this default. I know it's a public DNS server available to all, and since Google's not going anywhere, neither is this server. It just feels wrong to hard-code it...

That said, I don't have a better approach.